### DISCUSSION TITLE
Use symlinks with ddev service enable?

### DISCUSSION THREAD

**Discussion by deviantintegral** [ddev/ddev#4562] [Ideas]
One challenge with ddev services is that if a developers enables a service on their local, the service definition is copied. If the service configuration is changed, you have to manually remind any users to disable and re-enable the service to get the changes.

Is there any reason why `ddev service enable` doesn't use symlinks? I think switching that would be seen as a breaking change, but perhaps we could do `ddev service enable --link ...`? Though in practice I think I'm just going to recommend our team run `ln -s ...` to do it.

**Comment by rfay** [comment]
Seems like a good idea to me. Related:
 * https://github.com/drud/ddev/issues/3555

Also a near-term plan to do several other things in that area, https://github.com/drud/ddev/issues?q=is%3Aissue+is%3Aopen+label%3A%22addon+mechanism%22
